services/web/server/src/simcore_service_webserver/projects/_projects_service_delete.py

```diff
@@ -67,6 +67,7 @@ async def delete_project_as_admin(
     try:
         # 1. hide
         with _monitor_step(state, name="hide"):
```

Comment:
I am not sure I agree with you here.
ok the hidden field is not part of your document. and I hope there is a ProjectLocked before this function is called?
Cause otherwise that means I could call that function while you have the project opened, and if the deletion/trashing goes right away, then the project would go down, and be deleted.
Maybe whether there is a lock before this should be double checked.